ccloud/beginner-cloud/docker-compose.yml

```diff
@@ -1,3 +1,15 @@
+# docker-compose supports environment variable substitution with the ${VARIABLE-NAME} syntax.
+# Environment variables can be sourced in a variety of ways.  One of those ways is through
+# a well known '.env' file located next to the docker-compose.yml file.  See the Docker
+# documentation for details: https://docs.docker.com/compose/environment-variables/#the-env-file
+# 
+# This feature is being used to parameterize some values within this file.  In this directory is also
+# a .env file, which is actually a symbolic link to <examples-root>/utils/config.env.  That file
+# contains values which get substituted here when docker-compose parses this file.
+#
+# If you'd like to view the docker-compose.yml file rendered with it's environment variable substituions
```

Comment:
```suggestion
# If you'd like to view the docker-compose.yml file rendered with its environment variable substituions
```

Reply:
resolved, thank you.  i should have paid better attention in grammar school